scripts/install_deps.sh

```diff
@@ -55,7 +55,7 @@ detect_linux_distro() {
         DISTRO=$(lsb_release -is)
     elif [ -f /etc/os-release ]; then
         # extract 'foo' from NAME=foo, only on the line with NAME=foo
-        DISTRO=$(sed -n -e 's/^NAME="\(.*\)\"/\1/p' /etc/os-release)
+        DISTRO=$(sed -n -e 's/^NAME="\?\(.*\)\"\?/\1/p' /etc/os-release)
```

Comment:
I would not have expected the `\` in front of the `?` or is it required by sed?

Reply:
It is required by grep, actually it is required by POSIX because of historical reasons.
The first versions of POSIX utilities which used regular expressions didn't support `?` so when they added support for it they decided to require the `\` in front to keep backwards compatibility. That's also the reason why there's backslashes before the parentheses in that same expression.

Reply:
I checked, and it returns `Debian GNU/Linux"` for me. It seems the `.` is not greedy enough.

My suggestion:
```suggestion
        DISTRO=$(sed -n -e 's/^NAME="\?\(.*\)\("\|$\)/\1/p' /etc/os-release)
```

Reply:
You're right, committed your suggestion.